docs/create_your_own_brick.rst

```diff
@@ -119,11 +119,11 @@ to the tag attributes of the variables, as shown below:
     >>> i2 = tensor.matrix('i2')
     >>> y = foo.apply(i1, i2)
     >>> theano.printing.debugprint(y)
-    Elemwise{identity} [id A] 'foo_apply_output'   
-     |Elemwise{add,no_inplace} [id B] ''   
-       |Elemwise{identity} [id C] 'foo_apply_input1'   
+    Elemwise{identity} [id A] 'foo_apply_output'
+     |Elemwise{add,no_inplace} [id B] ''
+       |Elemwise{identity} [id C] 'foo_apply_input1'
        | |i1 [id D]
-       |Elemwise{identity} [id E] 'foo_apply_input2'   
+       |Elemwise{identity} [id E] 'foo_apply_input2'
```

Comment:
Removing this will probably result in the doctests failing because Theano actually prints trailing whitespace.